File: examples/python/core/wait/multiple-named-waits.py
# Good - clear purpose
context.wait(duration=Duration.from_seconds(60), name="rate_limit_cooldown")
context.wait(duration=Duration.from_minutes(5), name="polling_interval")

# Less clear - unnamed waits
context.wait(duration=Duration.from_seconds(60))
context.wait(duration=Duration.from_minutes(5))

File: examples/java/core/wait/multiple-named-waits.java
// Good - clear purpose
context.wait("rate_limit_cooldown", Duration.ofSeconds(60));
context.wait("polling_interval", Duration.ofMinutes(5));

// Less clear - unnamed waits
context.wait(null, Duration.ofSeconds(60));
context.wait(null, Duration.ofMinutes(5));
